src/style.css

```
@@ -0,0 +1,23 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.title {
+  font-size: 17px;
+  margin: 0;
+}
+
+.paragraph {
+  margin: 0;
+}
+
+.article {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+.link {
```

```suggestion
.colored-link {
```
додамо трохи конкретики

---

перейменувала